CAM Platform/workspaces.md

```diff
@@ -13,812 +13,892 @@ Manage and perform actions on workspaces.
-| name | string | Workspace name. |
-| created | string | Creation date. Example “2015-07-02 10:23:47.748740” |
-| uri | string | Workspace uri. |
-| members | array | Lists members of a team workspace. |
+| add_provider | boolean | Indicates true if a personal workspace has a provider |
+| billing_notice | boolean | If the flag is False we don't show the billing notice to the user |
+| clc_alias | string | (optional) Account Id for billing customers that have an account at CL |
+| clc_username | string | If user has CLC authentication |
+| created | string | Creation date |
+| costcenter | string | Cost center id |
+| dashboard_notice | boolean | If the flag is False we don't show the dashboard notice to the user |
+| deleted | boolean | If the workspace has been deleted |
+| deploy_instance | boolean | Indicates if the workspace has deployed an instance |
+| email | string | User email |
+| email_validated_at | string | Email validation date. |
+| favourites | array | List of user's favourite workspaces. User set a workspace as favourite when he clicks on a star. |
 | group_dns | array | List of fully qualified names of LDAP groups to which a user’s personal workspace belongs. You can’t update this field. Present in Personal Workspaces |
+| icon | string | Workspace icon |
+| id | string | Workspace unique identifier |
+| last_login | string | Date of the last login. Example “2015-07-02 10:23:47.748740” |
+| last_name | string | User last name |
 | ldap_groups | array | List of fully qualified names of LDAP groups that are members of a workspace. Present in Team Workspaces |
-| id | string | Workspace unique identifier. |
-| add_provider | xsd:boolean | Indicates true if a personal workspace has a provider. |
-| deploy_instance | xsd:boolean | Shows true when there are deployed instances in the personal workspace. |
-| email | string | User email, this parameter is used only in personal workspaces. |
+| members | array | Lists members of a team workspace |
+| name | string | User (Workspace) name |
+| organization | string | (Personal Workspaces) Organization of the workspace |
+| organizations | array | (Team Workspaces) List of the organizations' ids of the workspace |
 | owner | string | Refers to the username that owns the workspace. Present in Team Workspaces |
-| icon | string | Workspace icon. |
-| schema | string | Schema URI. Either “http://elasticbox.net/schemas/workspaces/personal” or “http://elasticbox.net/schemas/workspaces/team” |
+| saml_id | string | (optional) Account id if user logged in with SAML |
+| schema | string | Schema URI. For personal workspace:  “http://elasticbox.net/schemas/workspaces/personal” For team workspace:  “http://elasticbox.net/schemas/workspaces/team” |
+| support_user_created | boolean | If true, it is a support system user |
+| take_tour | boolean | If true, a hekp wizard will be displayed when users access the application |
```

Comment:
It's the other way around: `true` if the user has dismissed the tour pop up. Also, please correct `help`.